dss-api.yml

```diff
@@ -176,10 +188,22 @@ paths:
               type: string
               pattern: "^[a-z0-9]{64}$"
     put:
-      summary: Create a file
+      summary: Create a new version of a file
       description: |
-        Create a new file with a given UUID.  The file content is passed in through a cloud URL.  The file on the cloud
-        provider must have metadata set reflecting the file checksums and the file content type.
+        Create a new version of a file with a given UUID.  The file content is passed in through a cloud URL.  The file
+        on the cloud provider must have metadata set reflecting the file checksums and the file content type.
+
+        The metadata fields required are:
+        - hca-dss-content-type: content-type of the file
+        - hca-dss-sha256: SHA-256 checksum of the file
+        - hca-dss-sha1: SHA-1 checksum of the file
+        - hca-dss-s3_etag: S3 ETAG checksum of the file.  See
```

Comment:
We may want to check that these docstrings render properly as Markdown, as they will be consumed in autodocs downstream. Does Swagger UI render Markdown formatting?